### PR TITLE
Dev

### PR DESCRIPTION
**Pull request by Jeeremic**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="shortcut icon" href="/starter-code/assets/favicon-32x32.png" type="image/x-icon"> 
+    <link rel="stylesheet" href="style.css">
+    <title>Twix 3</title>
+</head>
+<body>
+    <div class="container">
+        <div class="logo-top">
```
Comment (ghost) [inline index.html:12]: Try using semantic HTML.

File: style.css [hunk 2]
```diff
@@ -0,0 +1,388 @@
+@import url('https://fonts.googleapis.com/css2?family=Red+Hat+Display:wght@500;600;900&display=swap');
+
+body {
```
Comment (ghost) [inline style.css:3]: If you take a closer look you have a margin around the whole page.

**Comment by Jeeremic** [comment]
Postovana Sandra,

ispravio sam margin i imacu u vidu ubuduce da ne koristim samo div-ove.
Hvala na savetima!

Pozdrav,
Stefan Jeremic.

**Comment by ghost** [comment]
Try to use English. :) You're welcome.

**Comment by ghost** [comment]
image
This is unnecessary space. Fix it.

**Comment by Jeeremic** [comment]
Sorry, I accidentally merged to master. But now I push changed

**Pull request merged**